doc/nrf/app_dev/device_guides/nrf54l/building_nrf54l.rst

```diff
@@ -0,0 +1,127 @@
+.. _building_nrf54l:
+
+Building and programming with nRF54L15 DK
+#########################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This guide provides instructions on how to build and program the nRF54L15 development kit.
+Whether you are working with single or multi-image builds, the following sections will guide you through the necessary steps.
+
+Depending on the sample, you must program only the application core or both the Fast Lightweight Peripheral Processor (FLPR) and the application core.
+Additionally, the process will differ based on whether you are working with a single-image or multi-image build.
+
+Building for the application core
+*********************************
+
+For instructions on building for the application core only, see how to :ref:`program an application <programming_cmd>`.
```

Comment:
This link points to https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/app_dev/programming.html which does not give any information about how to build. Link to `building` instead.

Again, does this mean that if I use a single-core build, I can use the general procedure? I'd suggest this is stated, as per my first comment.

Also, seeing the following sections: do I build with or without sysbuild? See one of my comments later.

Reply:
If you only need an app core image, you should use standard procedure (default is with sysbuild).